packages/apollo-server-express/src/__tests__/ApolloServer.test.ts

```diff
@@ -185,6 +185,57 @@ describe('apollo-server-express', () => {
       });
     });
 
+    it('renders GraphQL playground using request original url', async () => {
+      const nodeEnv = process.env.NODE_ENV;
+      delete process.env.NODE_ENV;
+      const samplePath = '/innerSamplePath';
+
+      const rewiredServer = new ApolloServer({
+        typeDefs,
+        resolvers,
+      });
+      const innerApp = express();
+      rewiredServer.applyMiddleware({ app: innerApp });
+
+      const outerApp = express();
+      outerApp.use(samplePath, innerApp);
+
+      const httpRewiredServer = await new Promise<http.Server>(resolve => {
```

Comment:
I'm not asking but if you're so compelled, a follow-up PR that [DRYs](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) up these tests would sure be very much welcomed.